docs/install/query.md

```diff
@@ -0,0 +1,41 @@
+# Run your first query using RubiX
+
+## Start Hive Client
+
+    hive --hiveconf hive.metastore.uris="" --hiveconf fs.rubix.impl=com.qubole.rubix.hadoop2.CachingNativeS3FileSystem
+
+## Create External Table
+
+    CREATE EXTERNAL TABLE wikistats_orc_rubix 
+    (language STRING, page_title STRING,
+    hits BIGINT, retrived_size BIGINT)
+    STORED AS ORC
+    LOCATION 'rubix://emr.presto.airpal/wikistats/orc';
+
+## Run Query (Presto or Hive CLI)
+
```

Comment:
I assume the syntax below is correct in both Presto and Hive.

Reply:
Yes. Verified